examples/experimental/tensorflow/mobilenet_v2/main.py

```diff
@@ -0,0 +1,162 @@
+    print('Checking the accuracy of the original model:')
+
+    metrics = [
+        tf.keras.metrics.CategoricalAccuracy(name='acc@1'),
+        tf.keras.metrics.TopKCategoricalAccuracy(k=5, name='acc@5')
+    ]
+
+    model.compile(metrics=metrics)
+    results = model.evaluate(data_source)
+    print(f'The original model accuracy@top1: {results[1]:.4f}')
+
+    print('Checking the accuracy of the quantized model:')
+    ie = ov.Core()
+    ir_model_xml = MODEL_DIR / f'{model_name}.xml'
+    ir_model_bin = MODEL_DIR / f'{model_name}.bin'
+    ir_quantized_model = ie.read_model(model=ir_model_xml, weights=ir_model_bin)
+    quantized_compiled_model = ie.compile_model(ir_quantized_model, device_name='CPU')
+    quantized_results = validate(quantized_compiled_model, metrics, data_source)
+    print(f'The quantized model accuracy@top1: {quantized_results[0]:.4f}')
+
+
+def create_data_source(batch_size: int) -> tf.data.Dataset:
+    """
+    Creates validation ImageNet dataset.
+
+    :param batch_size: A number of elements return per iteration.
+    :return: The instance of `tf.data.Dataset`.
+    """
+    val_dataset = tfds.load('imagenet2012', split='validation', data_dir=DATASET_DIR)
+
+    def preprocess(data_item: Dict[str, tf.Tensor]):
+        image = data_item['image']
+        image = center_crop(image, 224, 32)
+        image = tf.keras.applications.mobilenet_v2.preprocess_input(image)
+        image = tf.image.convert_image_dtype(image, tf.float32)
+
+        label = data_item['label']
+        label = tf.cast(label, tf.int32)
+        label = tf.one_hot(label, 1000)
+        label = tf.reshape(label, [1000])
+
+        return image, label
+
+    val_dataset = val_dataset.map(preprocess, num_parallel_calls=tf.data.AUTOTUNE)
+    val_dataset = val_dataset.batch(batch_size)
+
+    return val_dataset
+
+
+def validate(model: ov.Model,
+             metrics: List[Metric],
+             val_dataset: tf.data.Dataset,
+             print_freq: int = 100) -> Tuple[float]:
+    """
+    Validate the accuracy of the model.
+
+    :param model: An OpenVINO model.
+    :param metrics: A list of TensorFlow metrics
+    :param val_dataset: An instant of the TensorFlow dataset
+    :param print_freq: A print frequency (batch iterations).
+    :return: A Tuple of scalars of computed metrics
+    """
+    for m in metrics:
+        m.reset_state()
+
+    num_items = len(val_dataset)
+    for i, (images, labels) in enumerate(val_dataset):
+        input_data = images.numpy()
+
+        logit = model(input_data)
+        pred = list(logit.values())[0]
+
+        for m in metrics:
+            m.update_state(labels, pred)
+        if i % print_freq == 0 or i + 1 == num_items:
+            output = [f'{i + 1}/{num_items}:']
+            for m in metrics:
+                  output.append(f'{m.name}: {m.result().numpy():.4f}')
+            print(' '.join(output))
+
+    return metrics[0].result().numpy(), metrics[1].result().numpy()
```

Comment:
```suggestion
    return metrics[0].result().numpy().item(), metrics[1].result().numpy().item()
```
The return type is `Tuple[float]`.